steps/xcode-result-bundle-to-checks/1.0.0/step.yml

```diff
@@ -0,0 +1,72 @@
+title: Xcode Result Bundle to GitHub Checks
+summary: |
+  Generates a human-readable test report from the Xcode result bundle and shows it on GitHub Checks.
+description: |
+  Generates a human-readable test report from the Xcode result bundle and shows it on GitHub Checks.
+website: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks
+source_code_url: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks
+support_url: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks/issues
+published_at: 2021-10-25T19:38:24.734504+09:00
+source:
+  git: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks.git
+  commit: 5dfc36f022669c89faf63028cf91b41c005cbfa5
```

Comment:
tag `1.0.0` has to be moved to latest commit (after all changes are done) and hash here has to be updated to that hash. Or alternatively you can close this PR and re-open with all changes applied and the version tag moved to that commit.

Reply:
Fixed. Thanks for the review :D